sqlpage/templates/shell.handlebars

```diff
@@ -4,7 +4,9 @@
     <meta charset="utf-8"/>
     <title>{{default title "SQLPage"}}</title>
 
-    <link rel="stylesheet" href="/{{static_path 'sqlpage.css'}}">
+    <link rel="manifest" href="{{ site_prefix }}manifest.json">
```

Comment:
No we don't serve a manifest.json by default. I should make this an option to the shell I guess.

Is it a big deal if we are linking to a non existent file?
Full disclosure, I added the favicon link too.

Maybe I should raise a separate MR for that?

I think the favicon and the manifest are the only uses of the site_prefix helper so included them as example usage.

Reply:
Would you like me to raise a separate MR for the favicon and/or manifest links? I think they're good to have as it lets the user put a custom favicon even when on a different site_prefix.

Before, the browser would look for the favicon at /favicon.ico because no meta link in HTML. Hence it is somewhat related to the site_prefix feature in that setting a favicon for a site under a prefix is not currently possible without this change.

Of course, allowing the user to set any url as favicon would be even better, but at least for now placing a favicon or manifest at ``/site-prefix/favicon.ico`` or ``/site-prefix/manifest.json`` will actually work.

The introduction of a manifest.json is of course totally new so I get it if you'd rather a separate PR for that.

Reply:
Yes, let's handle the favicon and manifest separately. I think they should be properties of the shell component.

Reply:
Makes sense. Looks like you already removed those <link> tags in a016cdb. Resolving this thread